### DISCUSSION TITLE
Execute thymeleaf template from project directory

### DISCUSSION THREAD

**Discussion by testinging1** [thymeleaf/thymeleaf#852] [Q&A]
I created a library which, in combination with thymeleaf (without spring), deals with sending e-mails.
I add this library to my project dependencies and there is a problem - the library does not use the html template from the project's "resources / templates",it uses template from library resources.
How can this be changed? Do you have a sample solution?
I want this library to take html template for email from resources/templates of project that is using this library.
I tried with setting prefix to ClassLoaderTemplateResolver - every version of `file:src/main/resources/templates/` and classpath:src/main/resources/templates/ and so on, and so on. 
Which template resolver I should use and how to set it proper? Ty in advance for help!

**Comment by ultraq** [comment]
This question looks exactly the same as https://github.com/thymeleaf/thymeleaf/discussions/852, which has an answer in it there.
